core/src/test/scala/unit/kafka/utils/TestUtils.scala

```diff
@@ -808,10 +808,10 @@ object TestUtils extends Logging {
     * @param msg error message
     * @param waitTimeMs maximum time to wait and retest the condition before failing the test
     * @param pause delay between condition checks
-    * @param maxRetries maximum number of retries to check the given condition if a retriable exception is thrown
+    * @param retryExceptions whether to retry RetriableExceptions
     */
   def waitUntilTrue(condition: () => Boolean, msg: => String,
-                    waitTimeMs: Long = JTestUtils.DEFAULT_MAX_WAIT_MS, pause: Long = 100L, maxRetries: Int = 0): Unit = {
+                    waitTimeMs: Long = JTestUtils.DEFAULT_MAX_WAIT_MS, pause: Long = 100L, retryExceptions: Boolean = false): Unit = {
```

Comment:
Should this be called `retryRetriableExceptions`? Since it doesn't retry all exceptions. I do wonder if this is a good thing to have though. It seems like it's generally a bug if we're relying on this.

Reply:
All uses of the retries are for consumer group tests, which I've now maxed out the retries for. So I agree, this probably isn't necessary any more. I've removed the retry logic entirely.